<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="MayurJain" failures="1" tests="13" name="Default test" time="2.442" errors="0" timestamp="2023-07-01T14:28:56 GMT+05:30">
  <testcase classname="soucedemo.TestClasses.Practice" name="testA" time="0.031"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testA" time="0.007"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testA" time="0.009"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="loginTest" time="0.005"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="productAddToCart" time="0.005"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testD" time="0.005"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testE" time="0.008"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testF" time="2.034">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method soucedemo.TestClasses.Practice.testF() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method soucedemo.TestClasses.Practice.testF() didn't finish within the time-out 2000
at java.base@18/java.lang.Thread.sleep(Native Method)
at app//soucedemo.TestClasses.Practice.testF(Practice.java:68)
at java.base@18/java.lang.invoke.LambdaForm$DMH/0x0000000800d15c00.invokeVirtual(LambdaForm$DMH)
at java.base@18/java.lang.invoke.LambdaForm$MH/0x0000000800d18400.invoke(LambdaForm$MH)
at java.base@18/java.lang.invoke.Invokers$Holder.invokeExact_MT(Invokers$Holder)
at java.base@18/jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(DirectMethodHandleAccessor.java:154)
at java.base@18/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base@18/java.lang.reflect.Method.invoke(Method.java:577)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@18/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@18/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@18/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@18/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- testF -->
  <testcase classname="soucedemo.TestClasses.Practice" name="Btest" time="0.009"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testB" time="0.011"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="Atest" time="0.006"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="Dtest" time="0.014"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="Ctest" time="0.014"/>
  <testcase classname="soucedemo.TestClasses.Practice" name="testC" time="0.0">
    <ignored/>
  </testcase> <!-- testC -->
</testsuite> <!-- Default test -->
